main.go

```diff
@@ -165,6 +172,15 @@ func writeNewProfile(credFile *string, profileName *string, sourceProfile *strin
 	}
 }
 
+// printNewProfile prints out the commands to set the credentials as env variables
+// Returns nothing
+func printNewProfile(sessionDetails *sts.GetSessionTokenOutput) {
```

Comment:
I'm pretty sure this only works for bash. I think we'll want to detect the OS or shell or something to determine the right format.

Reply:
how about two options one for powershell/windows and one for bash? Id hate to do autodetect of the OS as I might need it for a different OS then I'm running on

Reply:
I'm ok with that. We'd need 3 formats then. One for Powershell, One for legacy CMD, and one for bash. so what if we did something like ```-env <format>``` where format is powershell|batch|bash? We could default to bash, or use autodetection to choose the default?

Reply:
So doing it in that way will take a bit longer than I would like to spend on this today :P I added another flag that defaults to bash but that you can set to cmd or pwshell to get those formats

Reply:
looks good. you can simplify the powershell ones:
```$env:AWS_...="value"``` but what you have technically works